### ISSUE TITLE
Use Eth1 address withdrawal by default

### ISSUE THREAD

**Issue by hwwhww** [ethereum/staking-deposit-cli#257]
On specs/EIPs upgrades side, the https://github.com/ethereum/consensus-specs/issues/2758 withdrawal feature is a promising and useful withdrawal approach after The Merge. It is reasonable to encourage the new validators to use Eth1 address withdrawal rather than BLS withdrawal now.

What to do:
- In depoist-cli, add prompt question "What's the withdrawal Ethereum address?"
- Add `--use-bls-withdrawal` flag that is only used in the command.
    - If this flag is set, then `--eth1_withdrawal_address` must be unset
- Update the instructions on launchpad

**Comment by djrtwo** [comment]
Agreed that this and other tools should make this shift. 0x01 addresses have many more safe custody options, features (e.g. smart contracts with key rotation), and will default support withdrawal features

**Comment by Kingtucks93** [comment]
--sync-mode=X_SNAP

**Comment by ptn77** [comment]
> On specs/EIPs upgrades side, the [ethereum/consensus-specs#2758](https://github.com/ethereum/consensus-specs/issues/2758) withdrawal feature is a promising and useful withdrawal approach after The Merge. It is reasonable to encourage the new validators to use Eth1 address withdrawal rather than BLS withdrawal now.
> 
> What to do:
> 
> * In depoist-cli, add prompt question "What's the withdrawal Ethereum address?"
> * Add `--use-bls-withdrawal` flag that is only used in the command.
>   
>   * If this flag is set, then `--eth1_withdrawal_address` must be unset
> * Update the instructions on launchpad

Is this the launchpad? I do not see the updates on the withdraw address options: https://launchpad.ethereum.org/en/generate-keys
